scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -589,6 +589,17 @@ def load_from_file(self, file, filter_platform=[]):
                     instance.status = status
                     instance.reason = reason
 
+                if self.quarantine:
+                    matched_quarantine = self.quarantine.get_matched_quarantine(
+                        instance.testsuite.id, platform.name, platform.arch, platform.simulation
+                    )
+                    if matched_quarantine and not self.options.quarantine_verify:
+                        instance.run = False
+                        instance.add_filter("Quarantine: " + matched_quarantine, Filters.QUARANTINE)
+                    if not matched_quarantine and self.options.quarantine_verify:
+                        instance.run = False
+                        instance.add_filter("Quarantine: " + matched_quarantine, Filters.QUARANTINE)
```

Comment:
this case if to run only quarantined tests (used with option `--quarantine-verify`). So you can keep in log "Not under quarantine" as is in `apply_filters` method

Reply:
the default one has to build first, which means we need have build envrionement on run envrionment, which is a problem I am tring to resolve here